app/liquid/views/partials/_form.liquid

```diff
@@ -10,6 +10,8 @@
         {% case field.data_type %}
         {% when 'text' or 'postal' %}
           <input type="text" name="{{field.name}}" class="form-control form__content" id="" placeholder="{{ field.label }}" maxlength="249" />
+        {% when 'hidden' %}
+          <input type="hidden" name="{{field.name}}" class="form-control form__content" id="" placeholder="{{ field.label }}">
```

Comment:
Hey Paul - I still haven't pushed up the default value stuff. It was almost done, but needed a little more before it was ready, and the rest of the stuff was more urgent for Joe to be able to use. I think it'll be a lot clearer when the default value stuff is implemented.